test/TestApps/OAuthTestAppPackage/OAuthTestAppPackage.wapproj

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is the purpose of this .wapproj file?

Reply:
A .wapproj file is a project file used in Visual Studio for Windows Application Packaging projects. The primary purpose of a .wapproj file is to define the configuration and settings needed to package a Windows application into an app package (such as an .msix or .appx file).